bot/classes/repository.js

```diff
@@ -32,10 +32,10 @@ class Repository extends RepositoryModel {
     }
 
     loadPluginsFromDisk() {
-        let pluginsPath = __dirname + "/../repositories/" + this._folderName + "/plugins";
+        let pluginsPath = "../repositories/" + this._folderName + "/plugins";
 
         if (fs.existsSync(pluginsPath)) {
-            fs.readdirSync(__dirname + "/../repositories/" + this._folderName + "/plugins").forEach(file => {
+            fs.readdirSync("../repositories/" + this._folderName + "/plugins").forEach(file => {
```

Comment:
I'm not sure changing the __dirname here is a good thing, my initial plan was to remove it from the require() since it's not necessary there.